encoder.py

```diff
@@ -107,22 +113,19 @@ def _get_pooling(self, feature_map: 'np.ndarray') -> 'np.ndarray':
         return self.pool_fn(feature_map, axis=(2, 3))
 
     @requests
-    def encode(self, docs: Optional[DocumentArray], parameters: Dict, **kwargs):
+    def encode(self, docs: Optional[DocumentArray], parameters: Optional[Dict] = None, **kwargs):
```

Comment:
parameters is not Optional

Reply:
Fixed in https://github.com/jina-ai/executor-image-torch-encoder/pull/4/commits/99541b46497b695f24e3db23495950d9fe614b9f